admin-ui/plugins/auth-server/redux/sagas/ScriptSaga.js

```diff
@@ -0,0 +1,77 @@
+import { call, all, put, fork, takeLatest, select } from 'redux-saga/effects'
+import { isFourZeroOneError } from 'Utils/TokenController'
+import {
+  getScriptsResponse,
+  getScriptsByTypeResponse,
+  getScriptResponse,
+} from '../features/scriptSlice'
+import { getAPIAccessToken } from 'Redux/features/authSlice'
+import ScriptApi from '../api/ScriptApi'
+import { getClient } from 'Redux/api/base'
+import * as JansConfigApi from 'jans_config_api'
+
+function* newFunction() {
+  const token = yield select((state) => state.authReducer.token.access_token)
+  const issuer = yield select((state) => state.authReducer.issuer)
+  const api = new JansConfigApi.CustomScriptsApi(getClient(JansConfigApi, token, issuer))
+  return new ScriptApi(api)
+}
```

Comment:
Rename helper for clarity.

newFunction is non-descriptive. Name it getScriptApi and update call sites.

```diff
-function* newFunction() {
+function* getScriptApi() {
   const token = yield select((state) => state.authReducer.token.access_token)
   const issuer = yield select((state) => state.authReducer.issuer)
   const api = new JansConfigApi.CustomScriptsApi(getClient(JansConfigApi, token, issuer))
   return new ScriptApi(api)
 }
```

```diff
-    const api = yield* newFunction()
+    const api = yield* getScriptApi()
```

```diff
-    const api = yield* newFunction()
+    const api = yield* getScriptApi()
```

```diff
-    const api = yield* newFunction()
+    const api = yield* getScriptApi()
```


Also applies to: 22-22, 36-36, 51-51